ci-operator/step-registry/redhat-appstudio/rhtap-cli/redhat-appstudio-rhtap-cli-commands.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+
+export OPENSHIFT_API \
+  OPENSHIFT_PASSWORD \
+  NAMESPACE \
+  GITHUB__APP__ID \
+  GITHUB__APP__CLIENT__ID \
+  GITHUB__APP__CLIENT__SECRET \
+  GITOPS__GIT_TOKEN \
+  GITHUB__APP__WEBHOOK__SECRET \
+  GITLAB__TOKEN \
+  GITLAB__APP__CLIENT__ID \
+  GITLAB__APP__CLIENT__SECRET \
+  QUAY__DOCKERCONFIGJSON \
+  QUAY__API_TOKEN \
+  ACS__CENTRAL_ENDPOINT \
+  ACS__API_TOKEN
+
+
+OPENSHIFT_API="$(yq e '.clusters[0].cluster.server' $KUBECONFIG)"
+NAMESPACE=rhtap
+GITHUB__APP__ID=$(cat /usr/local/rhtap-ci-secrets/rhtap/rhdh-github-app-id)
+GITHUB__APP__CLIENT__ID=$(cat /usr/local/rhtap-ci-secrets/rhtap/rhdh-github-client-id)
+GITHUB__APP__CLIENT__SECRET=$(cat /usr/local/rhtap-ci-secrets/rhtap/rhdh-github-client-secret)
+GITHUB__APP__PRIVATE_KEY=$(base64 -d < /usr/local/rhtap-ci-secrets/rhtap/rhdh-github-private-key | sed 's/^/        /')
+GITOPS__GIT_TOKEN=$(cat /usr/local/rhtap-ci-secrets/rhtap/gihtub_token)
+GITHUB__APP__WEBHOOK__SECRET=$(cat /usr/local/rhtap-ci-secrets/rhtap/rhdh-github-webhook-secret)
+GITLAB__TOKEN=$(cat /usr/local/rhtap-ci-secrets/rhtap/gitlab_token)
+GITLAB__APP__CLIENT__ID=$(cat /usr/local/rhtap-ci-secrets/rhtap/gitlab_oauth_client_id)
+GITLAB__APP__CLIENT__SECRET=$(cat /usr/local/rhtap-ci-secrets/rhtap/gitlab_oauth_client_secret)
+QUAY__DOCKERCONFIGJSON=$(cat /usr/local/rhtap-ci-secrets/rhtap/rhtap_quay_ci_token)
+QUAY__API_TOKEN=$(cat /usr/local/rhtap-ci-secrets/rhtap/quay_api_token)
+ACS__API_TOKEN=$(cat /usr/local/rhtap-ci-secrets/rhtap/acs-api-token)
+ACS__CENTRAL_ENDPOINT=$(cat /usr/local/rhtap-ci-secrets/rhtap/acs-central-endpoint)
+
+yq -i 'del(.clusters[].cluster.certificate-authority-data) | .clusters[].cluster.insecure-skip-tls-verify=true' "$KUBECONFIG"
+OPENSHIFT_PASSWORD="$(cat $KUBEADMIN_PASSWORD_FILE)"
+
+timeout --foreground 5m bash  <<- "EOF"
+    while ! oc login "$OPENSHIFT_API" -u kubeadmin -p "$OPENSHIFT_PASSWORD" --insecure-skip-tls-verify=true; do
+            sleep 20
+    done
+EOF
+
+if [ $? -ne 0 ]; then
+  echo "Timed out waiting for login"
+  exit 1
+fi
+
+configure_rhtap(){
+
+  echo "make build"
+  make build
+
+  # Path to your values.yaml.tpl file
+  tpl_file="charts/values.yaml.tpl"
+
+  # Turn ci to true
+  sed -i 's/ci: false/ci: true/' $tpl_file
+  
+  config_file="config.yaml"
+  sed -i '/redHatAdvancedClusterSecurity:/,/namespace: rhtap-acs/ s/^\(\s*enabled:.*\)$/#\1/' $config_file
+  sed -i '/redHatQuay:/,/namespace: rhtap-quay/ s/^\(\s*enabled:.*\)$/#\1/' $config_file
```

Comment:
@ascerra you need to add a sed command here as well change the catalog url for rhdh. It is still pointing to a release branch of the same templates which is point to a 1.0.x branch of the sample pipelines. I think that is why it is not picking up the fix for the promotion pipeline and still seeing that validation error. 

I would change the catalog url to https://github.com/redhat-appstudio/tssc-sample-templates/blob/main/all.yaml for testing purposes for the time being. See if we can get past the errors.